test/e2e/run_networking_test.go

```diff
@@ -42,38 +42,37 @@ var _ = Describe("Podman run networking", func() {
 	})
 
 	It("podman run network connection with default bridge", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", ALPINE, "wget", "www.podman.io"})
-		session.Wait(90)
+		session := podmanTest.RunContainerWithNetworkTest("")
+		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with host", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", "--network", "host", ALPINE, "wget", "www.podman.io"})
-		session.Wait(90)
+		session := podmanTest.RunContainerWithNetworkTest("host")
+		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with default", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "default", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.RunContainerWithNetworkTest("default")
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
 
 	It("podman run network connection with none", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "none", ALPINE, "wget", "www.podman.io"})
+		session := podmanTest.RunContainerWithNetworkTest("none")
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(1))
-		Expect(session.ErrorToString()).To(ContainSubstring("wget: bad address 'www.podman.io'"))
+		if _, found := os.LookupEnv("https_proxy"); found {
+			Expect(session).Should(Exit(5))
+			Expect(session.ErrorToString()).To(ContainSubstring("Could not resolve proxy:"))
+		} else {
+			Expect(session).Should(Exit(6))
+			Expect(session.ErrorToString()).To(ContainSubstring("Could not resolve host: www.podman.io"))
+		}
 	})
 
 	It("podman run network connection with private", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "private", ALPINE, "wget", "www.podman.io"})
-		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(0))
-	})
-
-	It("podman run network connection with loopback", func() {
-		session := podmanTest.Podman([]string{"run", "--network", "host", ALPINE, "wget", "www.podman.io"})
```

Comment:
Nice catch: this is an unnecessary dup. But the test name ("loopback") suggests that it was intended to be an actual test. I don't know what "loopback" is, there's no documentation for it in `podman-run(1)`, and it fails if I try running it. 

This code is almost five years old, PR #176 in January 2018. I realize the answer is almost certainly "heck no" but I have to ask anyway: @baude, do you have any recollection whatsoever of what this code was intended to test?